node/full_node_integration_test.go

```diff
@@ -312,33 +311,52 @@ func testSingleAggreatorSingleFullNodeSingleLightNode(t *testing.T) {
 	var wg sync.WaitGroup
 	aggCtx, aggCancel := context.WithCancel(context.Background())
 	ctx, cancel := context.WithCancel(context.Background())
-	clientNodes := 2
-	nodes, _ := createNodes(aggCtx, ctx, clientNodes+1, false, &wg, t)
 
-	node1 := nodes[0]
-	node2 := nodes[1]
-	node3 := nodes[2]
+	num := 3
+	keys := make([]crypto.PrivKey, num)
+	for i := 0; i < num; i++ {
+		keys[i], _, _ = crypto.GenerateEd25519Key(rand.Reader)
+	}
+	dalc := &mockda.DataAvailabilityLayerClient{}
+	ds, _ := store.NewDefaultInMemoryKVStore()
+	_ = dalc.Init([8]byte{}, nil, ds, log.TestingLogger())
+	_ = dalc.Start()
+	sequencer, _ := createNode(aggCtx, 0, false, true, false, keys, &wg, t)
+	fullNode, _ := createNode(ctx, 1, false, false, false, keys, &wg, t)
 
-	require.NoError(node1.Start())
-	time.Sleep(2 * time.Second) // wait for more than 1 blocktime for syncer to work
-	require.NoError(node2.Start())
+	sequencer.(*FullNode).dalc = dalc
+	sequencer.(*FullNode).blockManager.SetDALC(dalc)
+	fullNode.(*FullNode).dalc = dalc
+	fullNode.(*FullNode).blockManager.SetDALC(dalc)
 
-	node3.conf.Light = true
-	require.NoError(node3.Start())
+	lightNode, _ := createNode(
```

Comment:
[nitpick] merge all separate lines for each argument into 1 line. don't need separate lines.

Reply:
done